jet-spring-boot-starter/gradle.properties

```diff
@@ -0,0 +1,2 @@
+version = 0.1-SNAPSHOT
```

Comment:
I know this is a first release, but shouldn't we release the artifact with same version as spring boot we aim to be compatible with? E.g. 2.2.0 for current 2.2.x Spring Boot releases.